### ISSUE TITLE
Use NLP

### ISSUE THREAD

**Issue by u-ways** [u-ways/xTractor#1]
For next year, use NLP to categorise similar workouts together.
Here is a quick guide: https://towardsdatascience.com/how-to-build-a-fast-most-similar-words-method-in-spacy-32ed104fe498

**Comment by smbkr** [comment]
What’s the use case? Count how many times a given user did a run or a climb?

**Comment by u-ways** [comment]
@smbkr Yeah! Also find out if they're more of a streaker (i.e. x (1/4), x (2/4) ...) Or if they have some eater Xs (i.e. they put x when they drink or eat something, like a big meal)

Basically more fun stats to collect by categorising as much X's as possible!
